### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wanda-carlson**
Desc

### CODE REVIEW

**Comment by henry-richard** [comment]
henry comments

**Comment by wanda-carlson** [comment]
@henry-richard unlabel documentation

File: README.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 This is a public repository managed by PaidPaper. The code is based in Python. It has several GitHub actions.
 
 Testing
+
+//comments
```
Comment (henry-richard) [inline README.md:7]:
```suggestion
//comments wrong
```

File: README.md [hunk 2]
```diff
@@ -3,3 +3,5 @@
 This is a public repository managed by PaidPaper. The code is based in Python. It has several GitHub actions.
 
 Testing
+
```
Comment (henry-richard) [inline README.md:6]: remove space

**Pull request merged**